benchmarks/flatten.py

```diff
@@ -5,9 +5,10 @@
 import arkouda as ak
 
 
-def time_flatten(N, trials):
+def time_flatten(size, trials):
     print(">>> arkouda flatten")
     cfg = ak.get_config()
+    N = size * cfg["numLocales"]
```

Comment:
Flatten is pretty slow today (partially the kernel, but mostly the input array creation using `stick` (https://github.com/Bears-R-Us/arkouda/issues/1039)).

Think we'll need to drop the problem size from `10**6` to `10**5` for flatten for now.